zng/type.go

```diff
@@ -374,3 +373,71 @@ func TypeID(typ Type) int {
 	}
 	return typ.ID()
 }
+
+func FormatType(typ Type) string {
```

Comment:
Delete this since it's unused and looks equivalent to `FormatTypeValue(EncodeTypeValue(typ))`?

Reply:
Yeah, and formatType has the typedefs bug that I fixed on the other side.  Good call.

Reply:
The type value is stored in the context so I think we should have a method on Context to avoid the extra encoding.  But if you don't have the context, no big deal to encode the type value.

Reply:
I think we can also simplify zson/formatter.go now... we'll do this later